src/app/functions/server/onerep.d.ts

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type ProfileData = {
+  first_name: string;
+  last_name: string;
+  city: string;
+  state: import("../../../utils/states").StateAbbr;
+  birth_date?: import("../../../utils/parse.js").ISO8601DateString;
+  phone_number?: import("../../../utils/parse.js").E164PhoneNumberString;
```

Comment:
nit: can we `import` these types above on L4-ish?
Seems like we're using:
- `import("../../../utils/states")` 1&times;
- `import("../../../utils/parse.js")` 6&times;
    - `ISO8601DateString` 5&times;
    - `E164PhoneNumberString` 1&times;